### PR TITLE
Expand bridging headers to AffirmClient.h & AffirmRequest.h for SPM support

### PR DESCRIPTION
**Pull request by epyifany**
We are requesting a change to support the Affirm SDK SPM integration. 

Currently, `AffirmClient.h` & `AffirmRequest.h` are not included in the bridging header `AffirmSDK/AffirmSDK.h`. This makes it difficult to subclass in a client app. We were able to subclass through the SDK cocoapods.

To solve this problem, we added symlinks to these files. We edited the bridging header to include these files in the Swift package. 

Also branching from [dbarden:remove_broken_symlink](https://github.com/dbarden/affirm-merchant-sdk-ios/tree/remove_broken_symlink)
The files SPM/AffirmProtocol.{h,m} point to non-existing files and generates a warning when integrating the AffirmSDK via SPM.

### CODE REVIEW

**Review by bf-affirm — APPROVED** [review]
LGTM!

**Comment by epyifany** [comment]
@bf-affirm @kevinbartels @dbarden-sg @AlexanderGuntermann  Thank you guys for the code review! 

It shows that **2 workflows awaiting approval
First-time contributors need a maintainer to approve running workflows.**

Would you be able to help identify how we could merge this in and release? Thanks

**Review by bf-affirm — APPROVED** [review]

**Pull request merged**
